tests/virtualization/universal/save_and_restore.pm

```diff
@@ -35,6 +35,16 @@ sub run_test {
         }
     }
 
+    start_guests();
+
+    record_info "SSH", "Check hosts are listening on SSH";
+    wait_guest_online($_) foreach (keys %virt_autotest::common::guests);
+
+    foreach my $guest (keys %virt_autotest::common::guests) {
+        assert_script_run("ssh root\@$guest 'touch /var/empty_temp_file'");
+        assert_script_run("virsh destroy $guest", 90);
```

Comment:
Why do we power the VM off? We did not do that before.

Reply:
After saving the state, the VMs are shutdown. I have to power on the VMs to create the temp file in them and after that the VMs need to be shutdown again for the virsh restore.

Reply:
`script_retry "nmap $_ -PN -p ssh | grep open", delay => 3, retry => 60 foreach (keys %virt_autotest::common::guests);`
was part of the original code, I must have mist it and only made the change 
 `wait_guest_online($_) foreach (keys %virt_autotest::common::guests);` to my edits... :/